packages/twenty-server/src/engine/twenty-orm/repository/workspace-select-query-builder.ts

```
@@ -126,24 +126,24 @@ export class WorkspaceSelectQueryBuilder<
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  override getRawOne<U = any>(): Promise<U | undefined> {
+  override async getRawOne<U = any>(): Promise<U | undefined> {
     try {
       this.validatePermissions();
 
-      return super.getRawOne();
+      return await super.getRawOne();
```

These async await changes could be quite dangerous/impactful (affect lots of areas). Are you sure this is the way to go?

---

We do not need await for getRawOne, getRawMany, and getCount because we can return an unresolved promise for the call-sites to await them.

However, we do need await for computeTwentyORMException because we need to return the Error object, not the Promise.

Removed the await pointed out in the comment and fixed.